crypto/types.go

```diff
@@ -42,8 +42,8 @@ const (
 	fieldSize = 48
 	//
 	// 1 for compressed, 0 for uncompressed - values should not be changed
-	uncompressed = 0
-	compressed   = 1
+	//uncompressed = 0
```

Comment:
The commented value could be used by those settings : https://github.com/onflow/flow-go/blob/tarak/6340-FeldmanVSS-spam/crypto/types.go#L48-L49

I commented the line to avoid the linter complaint. I think it's better to use `nolint`.